docs/index.md

```diff
@@ -4,86 +4,55 @@
 [![codecov](https://codecov.io/gh/Trymzet/dyvenia/branch/main/graph/badge.svg?token=k40ALkXbNq)](https://codecov.io/gh/Trymzet/dyvenia)
 ---
 
-**Documentation**: <a href="https://dyvenia.github.io/viadot/" target="_blank">https://dyvenia.github.io/viadot/</a>
+**Documentation**: <a href="https://dyvenia.github.io/viadot/tree/2.0" target="_blank">https://dyvenia.github.io/viadot/tree/2.0</a>
 
-**Source Code**: <a href="https://github.com/dyvenia/viadot" target="_blank">https://github.com/dyvenia/viadot</a>
+**Source Code**: <a href="https://github.com/dyvenia/viadot/tree/2.0" target="_blank">https://github.com/dyvenia/viadot/tree/2.0</a>
 
 ---
 
 A simple data ingestion library to guide data flows from some places to other places.
 
-## Structure
-This documentation is following the diátaxis framework.
-
 ## Getting Data from a Source
 
-Viadot supports several API and RDBMS sources, private and public. Currently, we support the UK Carbon Intensity public API and base the examples on it.
+Viadot supports several API and RDBMS sources, private and public. An example of the use of the UK Carbon Intensity API.
 
 ```python
 from viadot.sources.uk_carbon_intensity import UKCarbonIntensity
 
 ukci = UKCarbonIntensity()
 ukci.query("/intensity")
 df = ukci.to_df()
-df
-```
 
-**Output:**  
+print(df)
+```
 
-|      | from              | to                |   forecast |   actual | index    |
-| ---: | ------------------|:------------------|-----------:|---------:|:---------|
-| 0    | 2021-08-10T11:00Z | 2021-08-10T11:30Z |        211 |      216 | moderate |
+**Output:**
 
+|      | from              | to                | forecast | actual | index    |
+| ---: | :---------------- | :---------------- | -------: | -----: | :------- |
+|    0 | 2021-08-10T11:00Z | 2021-08-10T11:30Z |      211 |    216 | moderate |
 
-The above `df` is a python pandas `DataFrame` object. The above df contains data downloaded from viadot from the Carbon Intensity UK API.
+The above `df` is a pandas `DataFrame` object. It contains data downloaded by `viadot` from the Carbon Intensity UK API.
 
 ## Loading Data to a Source
-Depending on the source, viadot provides different methods of uploading data. For instance, for SQL sources, this would be bulk inserts. For data lake sources, it would be a file upload. We also provide ready-made pipelines including data validation steps using Great Expectations.
-
-An example of loading data into SQLite from a pandas `DataFrame` using the `SQLiteInsert` Prefect task:
+Depending on the source, `viadot` provides different methods of uploading data. For instance, for SQL sources, this would be bulk inserts. For data lake sources, it would be a file upload. For ready-made pipelines including data validation steps using `dbt`, see [prefect-viadot](https://github.com/dyvenia/prefect-viadot).
```

Comment:
prefect-viadot is private so there's not much point linking it here

Reply:
[📝 Improved index.md](https://github.com/dyvenia/viadot/pull/657/commits/b0b0cf1528eb3eca265bc5a7ee2873a365d3686d)